README.md

```diff
@@ -220,15 +229,76 @@ manually you can do so by opening a sql shell.
       --certs-dir=${HOME}/.cockroachdb/certs/clients/root \
       --host localhost
 
-#### 4a. Setup cms database:
+#### 4a. Setup PostgresSQL databasee
```

Comment:
These scripts require the user have openssl installed, right?  That should be noted in the documentation.

Reply:
I see, used cockroach setup scripts as example here, I will go ahead and update both.

Reply:
Hrrmm I thought I mentioned that somewhere 👀